charts/finrem-cos/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: finrem-cos
 apiVersion: v1
 home: https://github.com/hmcts/finrem-case-orchestration-service
-version: 0.0.8
+version: 0.0.9
```

Comment:
No need to bump Chart version as no chart files will be changed

Reply:
reverted 👍